doc/classes/Object.xml

```diff
@@ -1,33 +1,37 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Object" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Base class for all non-built-in types.
+		Base class for all objects in the engine.
```

Comment:
As I mentioned on RC, I'm not sure it's a good idea to use the word that you are trying to describe in its own description. Though the "Object" class in Godot and the concept of "object" in OOP are not the exact same thing, the reader may not be aware of that. Changing it for a better term might prove hard due to how basic it is. I'd suggest a slight adjustment at least:

```suggestion
		Base class for all object types in the engine.
```

Reply:
Or maybe just use "classes" since this is what we do in the full description.

```suggestion
		Base class for all other classes in the engine.
```